source_code/src/GUI/mini_gui_credentials_functions.h

```diff
@@ -36,4 +36,16 @@ uint16_t guiAskForLoginSelect(pNode* p, cNode* c, uint16_t parentNodeAddress, ui
 uint16_t favoriteSelectionScreen(pNode* p, cNode* c);
 uint16_t loginSelectionScreen(void);
 
+#ifdef ENABLE_CREDENTIAL_MANAGEMENT
+#define MGMT_ACTION_NB 4
```

Comment:
same for commenting the #defines here.
Also, could you change the MGMT_ prefix to avoid confusion?